### DISCUSSION TITLE
别名到阿里云，能否增加对Aliyun Video Previewer的支持？谢谢。

### DISCUSSION THREAD

**Discussion by redapricot** [AlistGo/alist#4949] [General]
目前的挂载情况如下：
路径：`/storage/aliyun` 挂载阿里云盘
路径：`/movie` 别名挂载到 `/storage/aliyun/电影` 。
通过 `/storage/aliyun/电影` 直接访问是可以调用 `Aliyun Video Previewer` 播放视频的。
但通过别名后的路径 `/movie` 则只有 `Video` 可选。

多人使用需要使用别名功能控制用户的可访问目录。
有些奇怪的视频格式无法直接通过 `Video` 播放，如果使用 `Aliyun Video Previewer` 就可以利用阿里云官方的转码完美避开这个问题。
这个功能可否支持一下呢？谢谢。

**Comment by kumu7y** [comment]
现在有解决方案了吗
